configure.in

```diff
@@ -1145,6 +1142,9 @@ PKG_CHECK_MODULES([FREETYPE2],  [freetype2],
 PKG_CHECK_MODULES([TAGLIB],  [taglib >= 1.8],
   [INCLUDES="$INCLUDES $TAGLIB_CFLAGS"; LIBS="$LIBS $TAGLIB_LIBS"],
   AC_MSG_ERROR($missing_library))
+PKG_CHECK_MODULES([ICU_UC], [icu-uc >= 54.1],
```

Comment:
Thanks for looking into this. Fortunately only ubuntu 12.04 has such an old version, which does not include a pc file, >= 14.04 have v52.x, which includes one. Your configure change could work for the old "legacy" case. Don't worry too much about it though, I'm sure we'll get it to work.